util/src/main/scala/kamon/logback/RequestIDConverter.scala

```diff
@@ -0,0 +1,14 @@
+package kamon.logback
+
+import ch.qos.logback.classic.pattern.ClassicConverter
+import ch.qos.logback.classic.spi.ILoggingEvent
+import kamon.Kamon
+import kamon.context.Key
+
+class RequestIDConverter extends ClassicConverter {
+  val RequestIDKey = Key.broadcastString("request-id")
+
+  override def convert(event: ILoggingEvent): String = {
+    Kamon.currentContext().get(RequestIDKey).getOrElse("undefined")
+  }
+}
```

Comment:
I guess that due the class loader magic happening with Play on development mode, Logback can't find this class in the classpath. I compiled it and put in `lib/request-id-converter.jar` so that it can be found and used.